src/digisos/skjema/personopplysninger/bankinfo/Bankinformasjon.tsx

```diff
@@ -182,24 +181,19 @@ const Bankinformasjon = () => {
                             maxLength={13}
                             bredde={"S"}
                         />
-                        <div
-                            className={"inputPanel " + (kontonummer.harIkkeKonto ? " inputPanel__checked" : " ")}
-                            onClick={(event: any) => onChangeCheckboks(event)}
-                        >
-                            <Checkbox
-                                id="kontakt_kontonummer_har_ikke_checkbox"
-                                name="kontakt_kontonummer_har_ikke_checkbox"
-                                checked={kontonummer.harIkkeKonto ? kontonummer.harIkkeKonto : undefined}
-                                onChange={(event: any) => onChangeCheckboks(event)}
-                                label={
-                                    <div>
-                                        {intl.formatHTMLMessage({
-                                            id: FAKTUM_KEY_KONTONUMMER + ".harikke",
-                                        })}
-                                    </div>
-                                }
-                            />
-                        </div>
+                        <CheckboksPanel
+                            id="kontakt_kontonummer_har_ikke_checkbox"
+                            name="kontakt_kontonummer_har_ikke_checkbox"
+                            checked={kontonummer.harIkkeKonto ? kontonummer.harIkkeKonto : undefined}
+                            onChange={(event: any) => onChangeCheckboks(event)}
+                            label={
+                                <div>
+                                    {intl.formatHTMLMessage({
+                                        id: FAKTUM_KEY_KONTONUMMER + ".harikke",
```

Comment:
Ordet "faktum" kan etter hvert endres til noe annet senere i en ny PR. I stedet for Faktum brukes betyr egentlig nå bare noe sånt som "base language key" eller "språkhovednøkkel".

Reply:
Fint å få ryddet opp i dette ja, men enig i at det hører hjemme i en egen PR. :)